ctapipe_io_lst/calibration.py

```diff
@@ -446,6 +446,12 @@ def time_lapse_corr(self, event, tel_id):
             lst.svc.pixel_ids,
         )
 
+        # Set zeros if data is negative after time lapse correction to create a pedestal table
```

Comment:
If the timelapse correction is the code introducing this problem, why not fix it in the time lapse calibration?